atomicapp/nulecule/base.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, id, specversion, graph, basepath, metadata=None,
             params (list): List of params for the Nulecule application
             config (dict): Config data for the Nulecule application
             namespace (str): Namespace of the current Nulecule application
+            cli (dict): CLI data
```

Comment:
I think this is the wrong way to go about it. We are already processing what we call `cli_answers` in the [cli]( https://github.com/projectatomic/atomicapp/blob/8c0805751570ee66d3f5d620679c9f9c88f8c68b/atomicapp/cli/main.py#L479-L485)

We do need to process data from the CLI, but I don't know if this is the right place. We should unify around this approach and what we are already doing.